egui_glow/src/painter.rs

```diff
@@ -500,12 +484,18 @@ impl Painter {
                     .flat_map(|a| a.to_array())
                     .collect();
 
-                self.upload_texture_srgb(delta.pos, image.size, &data);
+                self.upload_texture_srgb(delta.pos, image.size, delta.filter.into(), &data);
```

Comment:
Is the `.into()` necessary? What does `cargo clippy` think?

Reply:
Whoops, my bad. Fixed it.